WarriorsSnuggery/WarriorsSnuggery.csproj

```diff
@@ -16,7 +16,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="3.3.1" Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'" />
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="3.6.0" Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'" />
```

Comment:
While you're at it, could you please remove the condition as well? It breaks building releases.

Reply:
Done.